Formula/libzt.rb

```diff
@@ -0,0 +1,39 @@
+class Libzt < Formula
+  desc "ZeroTier: libzt -- An encrypted P2P networking library for applications"
+  homepage "https://www.zerotier.com"
+
+  url "https://download.zerotier.com/dist/sdk/libzt-1.3.0-source.tar.gz"
+  sha256 "0db5e1df43275d17162b986be29fa8c530695e5503f207d27739703730b5b23c"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "cmake", "--build", "."
+    system "make", "install"
+    cp "licenses/ZEROTIER-LICENSE.GPL-3", "#{prefix}/LICENSE"
+  end
+
+  def caveats
+    <<~EOS
+      Visit https://my.zerotier.com to create virtual networks and authorize devices.
```

Comment:
This looks like documentation that should be on github. It isn't specific to homebrew installs.

Reply:
Done